### PR TITLE
Revert "Revert "Update README.md""

### PR DESCRIPTION
**Pull request by fukinbubba**
Reverts fukinbubba/fukinbubba#2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ Here are some ideas to get you started:
 - 📫 How to reach me: ...
 - 😄 Pronouns: ...
 - ⚡ Fun fact: ...
--->
```
Comment (fukinbubba) [inline README.md:16]: FukinBubba

Comment (fukinbubba) [reply README.md:16]: Taking Time

**Comment by fukinbubba** [comment]
Reverting

**Review by FKNCOIN — COMMENTED** [review]
Fukinbubba

**Review by FKNCOIN — COMMENTED** [review]
K

**Review by FKNCOIN — COMMENTED** [review]
K

**Review by FKNCOIN — COMMENTED** [review]
MAGA

**Pull request merged**